science/gr-iridium/Portfile

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+categories          science comms
+platforms           darwin macosx
+license             GPL-3+
+maintainers         {@ra1nb0w irh.it:rainbow} openmaintainer
+description         Iridium burst detector and demodulator.
+long_description    ${description}
+
+github.setup muccc gr-iridium eeaf8157dd78d5394bd7feefa3a1a5dd67af8037
+version   20190722-[string range ${github.version} 0 7]
```

Comment:
I'm not a fan of adding the shortened checksum. Just use the date. If by some rare event it needs to updated twice in a single day, just add an 'a' at the end.

Reply:
We have discussed a few times about this with @michaelld and at the end we prefer to maintain the commit on the version.